### DISCUSSION TITLE
any way to implement vuex devtools time travel to debug apps use xstate ?

### DISCUSSION THREAD

**Discussion by noor-tg** [statelyai/xstate#1602] [Q&A]
hey, I am new to xstate . and my previous state management lib is vuex . there is time travel feature in vuex tab in vue devtools . that allow user to go to a specific state which return the app to that state. and you can inspect the application in that state to find a bug or to check some thing.
can some thing like this be implemented with xstate ? and maybe with @xstate/inspector ?

**Comment by davidkpiano** [comment]
Yes, general dev-tools integration is planned for v5.
